pjsip/src/pjsip/sip_transaction.c

```diff
@@ -1952,6 +1952,7 @@ static void send_msg_callback( pjsip_send_state *send_state,
             tsx_update_transport(tsx, send_state->cur_transport);
 
             /* Update remote address. */
+		pj_assert(tdata->dest_info.cur_addr < (sizeof(tdata->dest_info.addr.entry)/sizeof(tdata->dest_info.addr.entry[0])));
```

Comment:
Minor: please limit column width to max 80 (old convention, just for consistency).

Reply:
Done. Modified using PJ_ARRAY_SIZE.